shaded-acceptance-tests-dependencies/pom.xml

```diff
@@ -114,31 +106,20 @@
         </executions>
       </plugin>
 
+      <plugin>
+        <groupId>org.sonatype.plugins</groupId>
+        <artifactId>nexus-staging-maven-plugin</artifactId>
+        <version>${nexus-staging-maven-plugin.version}</version>
+        <configuration>
+          <!-- TODO: Should we indeed exclude those shaded-XXX modules from Maven Central? -->
+          <!-- Instruct not to publish this module to Maven Central -->
+          <skipStaging>true</skipStaging>
+        </configuration>
+      </plugin>
```

Comment:
Since this module is only used for testing, I suggest it's not worth publishing it in Maven Central? Do you agree? If so, then do you confirm that this property actually works to exclude it?

Reply:
Note, I have already put this in the `shaded-deps-parent` POM here: https://github.com/GoogleCloudDataproc/hive-bigquery-connector/blob/646bf2b44942f8391bb2d9e1ec0e58e2bacf8524/shaded-deps-parent/pom.xml#L149-L158

Let me know if it's ok, or if those should be removed in order to publish everything to Maven Central.

Reply:
I agree it is better to skip. Only the connector jar is needed.

Reply:
Ok cool!